src/__tests__/app.tsx

```diff
@@ -4,11 +4,15 @@
 
 // import React from 'react';
 // import ReactDOM from 'react-dom';
-// import App from '../App';
+
+// import App from '../app';
 
 describe('App Test Suite', () => {
   it('renders without crashing', () => {
-    // const div = document.createElement('div');
+    // document.createElement('div');
     // ReactDOM.render(<App />, div);
   });
 });
+
+// Empty export is required if there are no imports
+export { };
```

Comment:
That's a weird quirk of TypeScript, do you have any references as to why this needs to be done?

Reply:
Yeah, the precise error is: 

```
'app.tsx' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module. ts(1208)
```

[More info at typescriptlang.org](https://www.typescriptlang.org/tsconfig#isolatedModules)

Of note is, this error goes away if the test is not commented out. The problem with that, though, is the test does not pass.

Reply:
Potential fix for this may be to add another tsconfig in the test directory, like:

```json
{
  "extends": "../tsconfig.json",
  "compilerOptions": {
    "isolatedModules": false
  },
}
```

Reply:
Yah i'm used to use that export {} sometimes when used isolatedModules, and isolatedModules sometimes its required for some others code to build correctly using babel.

Reply:
Ah ok, this test seems to be faulty, so it should be fine now to just add and empty export to get things compiling.